### PR TITLE
Adjust tensor sizes to fix plasticity error

### LINKED ISSUE #984: off-fault plasticity is broken since #963
**Describe the bug**
The tpv13 from SeisSol/Training generates too large plastic strain since #963 was merged.

```
#before merging #963
0 Tue Oct 24 17:43:23, Info:  Writing faultoutput at time 0.
0 Tue Oct 24 17:43:23, Info:  Writing faultoutput at time 0. Done.
0 Tue Oct 24 17:43:23, Info:  Waiting for last wave field.
0 Tue Oct 24 17:43:23, Info:  Writing wave field at time 0.
0 Tue Oct 24 17:43:23, Info:  Writing wave field at time 0. Done.
0 Tue Oct 24 17:43:23, Info:  Writing free surface at time 0.
0 Tue Oct 24 17:43:23, Info:  Writing free surface at time 0. Done.
0 Tue Oct 24 17:43:23, Info:  Writing energy output at time 0
0 Tue Oct 24 17:43:23, Info:  Writing energy output at time 0 Done.
0 Tue Oct 24 17:43:24, Info:  Writing energy output at time 0.5
0 Tue Oct 24 17:43:24, Info:  Elastic energy (total, % kinematic, % potential):  7.56064e+14  , 34.8231  , 65.1769
0 Tue Oct 24 17:43:24, Info:  Plastic moment (value, equivalent Mw, % total moment): 1.47672e+16  , 4.70953  , 2.02751
0 Tue Oct 24 17:43:24, Info:  Frictional work (total, % static, % radiated):  7.69503e+14  , 30.8249  , 69.1751
0 Tue Oct 24 17:43:24, Info:  Seismic moment (without plasticity): 7.13571e+17  Mw: 5.83229
0 Tue Oct 24 17:43:24, Info:  Writing energy output at time 0.5 Done.
```
vs 

```
#after merging #963
0 Tue Oct 24 17:47:05, Info:  Starting simulation.
0 Tue Oct 24 17:47:05, Info:  Writing faultoutput at time 0.
0 Tue Oct 24 17:47:05, Info:  Writing faultoutput at time 0. Done.
0 Tue Oct 24 17:47:05, Info:  Waiting for last wave field.
0 Tue Oct 24 17:47:05, Info:  Writing wave field at time 0.
0 Tue Oct 24 17:47:05, Info:  Writing wave field at time 0. Done.
0 Tue Oct 24 17:47:05, Info:  Writing free surface at time 0.
0 Tue Oct 24 17:47:05, Info:  Writing free surface at time 0. Done.
0 Tue Oct 24 17:47:05, Info:  Writing energy output at time 0
0 Tue Oct 24 17:47:05, Info:  Writing energy output at time 0 Done.
0 Tue Oct 24 17:47:06, Info:  Writing energy output at time 0.5
0 Tue Oct 24 17:47:06, Info:  Elastic energy (total, % kinematic, % potential):  7.56064e+14  , 34.8231  , 65.1769
0 Tue Oct 24 17:47:06, Info:  Plastic moment (value, equivalent Mw, % total moment): 4.33939e+23  , 9.68829  , 99.9998
0 Tue Oct 24 17:47:06, Info:  Frictional work (total, % static, % radiated):  7.69503e+14  , 30.8249  , 69.1751
0 Tue Oct 24 17:47:06, Info:  Seismic moment (without plasticity): 7.13571e+17  Mw: 5.83229
0 Tue Oct 24 17:47:06, Info:  Writing energy output at time 0.5 Done.
```

```
#after merging #963
di73yeq4@login03:/hppfs/work/pr45fi/di73yeq4/Training/tpv13> python ~/TuSeisSolScripts/onHdf5/minmax_SeisSol_data.py output/tpv13.xdmf --Data eta
t eta eta
0.0 0.0 0.0
4.0 -15752.347848275807 361388.0169528889
8.0 -7278.0092620853775 1380203.6737889433
#state at last PR merged before #963
di73yeq4@login03:/hppfs/work/pr45fi/di73yeq4/Training/tpv13> python ~/TuSeisSolScripts/onHdf5/minmax_SeisSol_data.py output/tpv13.bak_2023-10-24_17-47-04.xdmf --Data eta
t eta eta
0.0 0.0 0.0
4.0 -0.001018839348114374 0.02044908173580127
8.0 -0.002961597842155997 0.03787899669585106
```

**To Reproduce**
1. Which version do you use? Provide branch and commit id.

after:fca0280e24c8ed7cc2e06e657b5f85182d49a5ac
before:7901b7c2964050c57b8cb84c3f510f05e99d7b53
(master)
2. Which build settings do you use? Which compiler version do you use? 
```
cmake -DCOMMTHREAD=ON -DNUMA_AWARE_PINNING=ON -DASAGI=ON -DCMAKE_BUILD_TYPE=Release -DHOST_ARCH=skx -DPRECISION=double -DORDER=4 -DGEMM_TOOLS_LIST=LIBXSMM,PSpaMM ..
```

3. On which machine does your problem occur? If on a cluster: Which modules are loaded?
supermucNG
```
Currently Loaded Modulefiles:
 1) admin/1.0     4) spack/22.2.1                                     7) intel-mpi/2019-intel  10) python/3.8.11-extended                
 2) tempdir/1.0   5) intel-oneapi-compilers/2021.4.0(default:intel)   8) gcc/11.2.0            11) numactl/2.0.14-intel21                
 3) lrz/1.0       6) intel-mkl/2020                                   9) cmake/3.21.4          12) seissol-env/develop-intel21-impi-x2b  

Key:
(symbolic-version)  default-version  sticky  
```

4. Provide parameter/material files.
see Training/tpv13

### PR DESCRIPTION
**Pull request by davschneller**
Fixes #984 . Also, we remove references to `NUMBER_OF_ALIGNED_BASIS_FUNCTIONS` and replace them with the proper tensor sizes, to avoid potential future errors.

### CODE REVIEW

**Comment by davschneller** [comment]
However, note that I'm still not super happy with how the plasticity kernel looks at the moment (and it's hard-wired to an elastic setup)—we directly access the arrays; maybe it's more useful to copy them using tensor initializers, in case the structure changes in the future, but maybe that'll not be a problem... We'll see.

File: src/Kernels/Plasticity.cpp [hunk 1]
```diff
@@ -172,7 +171,7 @@ namespace seissol::kernels {
       adjKrnl.execute();
 
       // calculate plastic strain with first dof only (for now)
```
Comment (Thomas-Ulrich) [inline src/Kernels/Plasticity.cpp:173]: this comment is outdated

**Review by Thomas-Ulrich — APPROVED** [review]
see nevertheless my comment (not related with the PR)

File: src/Kernels/Plasticity.cpp (hunk 1 above)
Comment (davschneller) [reply src/Kernels/Plasticity.cpp]: Ok, changed it to `// calculate plastic strain`

**Review by Thomas-Ulrich — APPROVED** [review]

**Pull request merged**